cpp/bench/prims/linalg/masked_matmul.cu

```diff
@@ -93,24 +102,49 @@ struct MaskedMatmulBench : public fixture {
     c_indices_d.resize(c_true_nnz, stream);
     c_dense_data_d.resize(params.m * params.n, stream);
 
-    cpu_convert_to_csr(bitmap_h, params.m, params.n, indices, indptr);
+    cpu_convert_to_csr(bits_h, params.m, params.n, indices, indptr);
     RAFT_EXPECTS(c_true_nnz == c_indices_d.size(),
                  "Something wrong. The c_true_nnz != c_indices_d.size()!");
 
     update_device(c_data_d.data(), values.data(), c_true_nnz, stream);
     update_device(c_indices_d.data(), indices.data(), c_true_nnz, stream);
     update_device(c_indptr_d.data(), indptr.data(), params.m + 1, stream);
-    update_device(bitmap_d.data(), bitmap_h.data(), element, stream);
```

Comment:
We want to support both bitmap and bitset inputs but it appears we're removing the bitmap support.

Reply:
Not true, `bits` is only a naming, these code is compatible with bitset and bitmap, so I need to change `bitmap` to `bits`, as compatible control point is[ here](https://github.com/rapidsai/raft/pull/2523/files/162a7415b6bb30816b9f46542bf4c2722ffdf64d#diff-39a056e37609051bc02fd448f48c7b5a7fb471c9a232a4e22d52803c03b39706R221)

Reply:
This is not easy to understand and it’s not obvious which one is selected when true or false (future eyes are going to be confused too) . Let’s create an enum for this that we can share across benchmarks and tests. It’ll make this more straightforward for future eyes too.